content/news/2021-08-19-gsoc-tools.md

```diff
@@ -0,0 +1,64 @@
++++
+title = "GSoC 2021: Tooling Final Summary"
+date = 2021-08-19
+extra.author = "Devin Ragotzy"
++++
+
+## Google Summer of Code Comes to an End for Ruma Tooling
```

Comment:
If the intro section doesn't need a heading that's much different from the title, maybe just remove that heading? The title will be right above.

Reply:
Should I change the first sentence from "What a fun summer!" to "What a fun Google Summer of Code!" (we talked about adding the full name to something in the beginning)?

Reply:
No, I think that sounds a bit weird. Let's just keep it the way it is. Worst case people will have to google GSoC ^^